modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/cell_renderers/DateTimeRenderer.js

```diff
@@ -22,7 +22,7 @@ function DateTimeRenderer({options, value}) {
 	};
 
 	if (
-		!dateOptions.timeZone &&
+		options?.format?.convertToUserTimeZone &&
```

Comment:
- On naming, maybe something like  `useThemeDisplayTimeZone`?
- Alternative to adding an explicit flag would be that `timeZone` has a special value, something like `ThemeDisplayTimeZone`. In the future, we could have other special values, like `BrowserTimeZone`. I don't have a strong opinion on this, either seems fine.

Reply:
Well, this is a possibility yep. It's true that `convertToUserTimeZone` flag overrides the `timezone` value that can be provided in the config, because renderer just ignores such value.

Putting all together into `timezone` makes it a bit more difficult to configure, because we need to add special values, but on the other side, the API to configure renderer becomes a bit more clear.

As we are exposing this API for both DSM admins and Java implementors, we want it to be simple.

I'm going to try this and see how it looks like.